docs/source/guide/labeling.md

```diff
@@ -248,25 +248,25 @@ You can add multiple types of regions to image annotations. You can add any of t
 To add different types of regions to your image annotations, follow this example.
 
 Create a custom template for your labeling interface using the following example:
+
 ```xml
 <View>
-  <Image name="image" value="$image" />
-  <Rectangle name="rect" toName="image" />
-  <Ellipse name="ellipse" toName="image" />
-  <KeyPoint name="kp" toName="image" />
-  <Polygon name="polygon" toName="image" />
-  <Brush name="brush" toName="image" />
-  <Choices name="choices" toName="image">
-    <Choice value="yes"></Choice>
-    <Choice value="no"></Choice>
-  </Choices>
-<Labels name="labels" toName="image" fillOpacity="0.5" strokeWidth="5">
-  <Label value="building" background="green"></Label>
-  <Label value="vehicle" background="blue"></Label>
-  </Labels>
+    <Image name="image" value="$image" />
```

Comment:
do we use 4 spaces indents in other places? I believe it's 2 everywhere

Reply:
Updated to use two spaces instead of 4